src/app/components/search/search.html

```diff
@@ -9,6 +9,14 @@
     .sub-results {
         margin-left: 36px;
     }
+    .line-clamp {
+	   overflow: hidden;
+	   text-overflow: ellipsis;
+	   display: -webkit-box;
+	   -webkit-line-clamp: 2; 
+	   -webkit-box-orient: vertical;
+	}
```

Comment:
According to https://stackoverflow.com/a/13924997, this works everywhere except for IE, which I can't imagine is a big consumer of dbt docs?